tests/baselines/reference/api/typescript.d.ts

```diff
@@ -5557,7 +5571,7 @@ declare namespace ts {
     }
     interface ClassifiedSpan {
         textSpan: TextSpan;
-        classificationType: ClassificationTypeNames;
+        classificationType: ClassificationTypeNames | number;
```

Comment:
I think this is going to be breaking change.. Everyone who uses this type (even if they arent using the new classification) are going to have to handle this and that doesnt seem necessary

Reply:
Why does this have to be number and not string just like before?

Reply:
If this has to be number then we need overloads that return the new type with number (should definitely use enum instead as what does number mean to API user ?) only if format is specified

Reply:
Great point, I can think I can make this not a breaking change with some overloads and a separate type. The number makes sense in context (one goal of the new system is to be much less chattier, so it sends something like 6701 instead of `class.declaration`)

Reply:
I took a stab at this in https://github.com/orta/TypeScript/pull/4 but haven't got it compiling yet

Reply:
Definitely needs enum in that case

Reply:
Calling it an `enum` instead of a `number` is tricky because it represents an encoded mix of both `ts.classifier.v2020.TokenType` and `ts.classifier.v2020.TokenModifier`.

I could make a new type like `type ClassifierTokenTypeSpan = number` which describes what it does?